### ISSUE TITLE
Is check integrity of boot medium feature of live USB not tailored to Feren OS files?

### ISSUE THREAD

**Issue by MystesofEternity** [feren-OS/Bug-Reporting-Center#41]
**Package Name or Section of Feren OS where this bug resides**
Check integrity of boot medium feature

**Describe the bug**
After booting with USB and using this feature, results came up and told me that there were errors on 9 files

**Steps to Reproduce**
1. Boot Feren OS using USB as a boot medium
2. When menu shows up, select check integrity of boot medium
3. Wait for results to show up and say that there were errors on 9 files or whatever number.

**Expected behavior**
Result should've showed up and say that there were no errors found within the boot medium.

**Additional context**
I made the boot medium through linux using the command
`sudo dd if=[name of Feren OS.iso] of=/dev/sdb bs=4M status=progress oflag=sync`
If there is a problem with using this command along with Feren OS please tell me.
There were no problems booting the medium and installing Feren OS but I am concerned that some things may be broken, I haven't extensively tested everything as of yet.

**Comment by feren** [comment]
Yeah, unfortunately right now it isn't tailored... which is weird as I have tried to change what I know to be file checks in there to tailor them to the Feren OS ISO.

**Comment by feren** [comment]
I've removed that option from the boot menu for now until I find a way to fix this one day.

**Comment by MystesofEternity** [comment]
Thank you very much for all the hard work!
